src/status_im/ui/screens/chat/views.cljs

```diff
@@ -285,8 +291,8 @@
         all-loaded? @(re-frame/subscribe [:chats/all-loaded? chat-id])]
     [react/view {:style (when platform/android? {:scaleY -1})}
      (if (or loading-messages? (not chat-id) (not all-loaded?))
-       [react/view {:height 324 :align-items :center :justify-content :center}
-        [react/activity-indicator {:animating true}]]
+       [react/view {:height @messages-view-height}
+        [messages-skeleton/messages-skeleton]]
```

Comment:
seems like it's better to pass height as a parameter to messages-skeleton and do not have extra view, also in that case you don't need `on-layout` twice

Reply:
Changed @flexsurfer 👍 . Passing the height from the parent. Please review.